File: Seminar5/HomeWork/View/ViewRational.java
package HomeWork.View;

import java.util.Scanner;

public class ViewRational {
    Scanner in = new Scanner(System.in);

    public double getValue(String title) {
        System.out.printf("%s", title);
        return in.nextDouble();
    }

    // print(f'{title} = {data}')
    public void print(double data, String title) {
        System.out.printf("%s %f\n", title, data);
    }

}

File: FinalTaskCalculator/View/ViewRational.cs
using System;

namespace View
{

    public class ViewRational
    {

        public double GetValue(String title)
        {
            Console.WriteLine(title);
            return Convert.ToDouble(Console.ReadLine());
        }

        public void Print(double data, String title)
        {
            Console.WriteLine(title + " " + data);
        }

    }
}